tests/dataset_histograms/histogram_error_estimator_test.py

```diff
@@ -49,9 +49,18 @@ def _get_estimator(
         epsilon: float = 2**0.5 / 2,
         delta: Optional[float] = None,
     ):
-        return histogram_error_estimator.create_error_estimator(
+        return histogram_error_estimator.create_estimator_for_count_privacy_id_count(
```

Comment:
I would rename from `_get_estimator` to `_get_estimator_for_count_and_privacy_id_count`.

Reply:
thx, done